requirements.txt

```diff
@@ -1,11 +1,13 @@
 cement==3.0.4
-pyyaml
+pyyaml~=5.3.1
 colorlog
 
-jinja2>=2.10
 setuptools>=38.6.0
 urllib3~=1.25.8
 confuse~=1.3.0
 certifi >= 14.05.14
 six >= 1.10
 python_dateutil >= 2.5.3
+
+pytest~=6.1.1
+xrdsst~=0.1.0
```

Comment:
This line from PR update causes now ``make virtualenv`` to fail, it should probably be simply removed.

Reply:
>     * When `xrdsst` is invoked without any arguments, I think it should not exit successfully, but instead print brief usage message (probably to `stderr`, not to `stdout`) and exit with non-zero code. Currently it is completely quiet and exits successfully -- when e.g. constructing `xrdsst` calls with user arguments in scripts, this can get error-prone.

I see this behaviour has remained the same, if it cannot be addressed now, we will need to keep this in mind.